README.md

```diff
@@ -103,6 +103,106 @@ If you are using the types in a `commonjs` module, like in a Node app, you just
 
 ## Trouble shooting and known issues.
 
+### Web3 and Create-react-app
+
+If you are using create-react-app version >=5 you may run into issues building. This is because polyfills are not included in the latest version of create-react-app.
+
+### Solution
+
+
+- Install react-app-rewired
+
+If you are using yarn:
+```bash
+yarn add --dev react-app-rewired
+```
+
+If you are using npm:
+```bash
+npm install --save-dev react-app-rewired
+```
+
+- Create `config-overrides.js` in the root of your project folder with the content:
+
+```javascript
+module.exports = function override(config) {
+    const fallback = config.resolve.fallback || {};
+    Object.assign(fallback, {
+        "crypto": require.resolve("crypto-browserify"),
+        "stream": require.resolve("stream-browserify"),
+        "assert": require.resolve("assert"),
+        "http": require.resolve("stream-http"),
+        "https": require.resolve("https-browserify"),
+        "os": require.resolve("os-browserify"),
+        "url": require.resolve("url")
+    })
+    config.resolve.fallback = fallback;
+
+    return config;
+}
+```
+
+
+- Install the missing modules
+
+If you are using yarn:
+```bash
+yarn add --dev crypto-browserify stream-browserify assert stream-http https-browserify os-browserify url
+```
+If you are using npm:
+```bash
+npm install --save-dev crypto-browserify stream-browserify assert stream-http https-browserify os-browserify url
+```
+
+- Create the file `polyfill.js` in the src folder and in the file add:
+
+```javascript
+import { Buffer } from 'buffer';
+
+window.global = window;
+global.Buffer = Buffer;
+global.process = {
+    env: { DEBUG: undefined },
+    version: '',
+    nextTick: require('next-tick')
+};
```

Comment:
I think we can do this with webpack ProviderPlugin. We don't need a separate script file.